src/main/java/org/isf/admission/manager/AdmissionBrowserManager.java

```diff
@@ -382,15 +376,15 @@ else if (dateOut.before(ad.getAdmDate()) || dateOut.equals(ad.getAdmDate())) {
 						OHSeverityLevel.ERROR));
 			}
 
-			GregorianCalendar visitDate = admission.getVisitDate();
-			if (visitDate != null && ward.getCode().equalsIgnoreCase("M")) {
-				GregorianCalendar limit;
+			LocalDateTime visitDate = admission.getVisitDate();
+			if (operationDate != null && ward.getCode().equalsIgnoreCase("M")) {
+				LocalDateTime limit;
```

Comment:
The hotfix here was:
```
-			if (operationDate != null && ward.getCode().equalsIgnoreCase("M")) {
+			if (visitDate != null && ward.getCode().equalsIgnoreCase("M")) {
```
so it seems this PR is rolling back the hotfix, isn't it?